src/transaction/check-cancel.ts

```diff
@@ -0,0 +1,32 @@
+import * as utils from './utils'
+import {validate} from '../common'
+import {Instructions, Prepare} from './types'
+
+type CheckCancel = {
+  checkID: string
+}
+
+function createCheckCancelTransaction(account: string,
+  cancel: CheckCancel
+): Object {
```

Comment:
`Object` actually doesn't mean an arbitrary object like it did in Glow, it's closer to an empty object than an arbitrary one. Change to `any` or `object` (lower-case) to represent any arbitrary object, and then you should be able to remove the `: any` you had to add to the txJSON below.